### ISSUE TITLE
openssl 3.0.2-0ubuntu1.10 CVEs

### ISSUE THREAD

**Issue by xtremerui** [paketo-buildpacks/jammy-base-stack#103]
## Expected Behavior
We are expecting to see the CVE scan of image that based on paketobuildpacks/run-jammy-base to not show CVE-2022-2068 and CVE-2022-1292

## Current Behavior
CVE scan of paketobuildpacks/run-jammy-base detects above two CRITICAL CVEs, based on NVD 

## Possible Solution
Bump openssl to 3.0.3

## Steps to Reproduce
```
docker run --rm -it --user root paketobuildpacks/run-jammy-base sh

# openssl version
OpenSSL 3.0.2 15 Mar 2022 (Library: OpenSSL 3.0.2 15 Mar 2022)
```

## Motivations
The compliance expects the product built on top of paketo image to not have any CRIT/HIGH CVE so we are hoping `openssl` could be bumped to a newer version to not have such CVEs.

**Comment by robdimsdale** [comment]
Unfortunately this is actually a false positive with the scanning tool not understanding how Ubunut/Canonical patches libraries.

If we look at: https://ubuntu.com/security/CVE-2022-1292 we see the following: 

<img width="835" alt="Screenshot 2023-07-10 at 21 37 38" src="https://github.com/paketo-buildpacks/jammy-base-stack/assets/7230694/bffdfb22-789a-4a6d-9780-fe424668e7eb">

---

Similarly if we look at https://ubuntu.com/security/CVE-2022-2068 we see:
<img width="825" alt="Screenshot 2023-07-10 at 21 37 52" src="https://github.com/paketo-buildpacks/jammy-base-stack/assets/7230694/9c1750a0-023a-45a0-ba92-fc6df3a65131">

---

You see that the versions are of the format: `3.0.2-0ubuntu1.x`. This means that Canonical maintains their own patches of `openssl`. So instead of bumping to `3.0.3` they backport the fixes from `3.0.3` (and potentially more fixes than make it into the upstream `openssl` distribution).

So as long as the version is `3.0.2-0ubuntu1.5` or greater, these CVEs are patched.

Looking on the latest run image (using your command from above) we see:

```
❯  docker run --rm -it --user root paketobuildpacks/run-jammy-base
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
root@702d34eced03:/# dpkg -l | grep -i openssl
ii  openssl                 3.0.2-0ubuntu1.9                        amd64        Secure Sockets Layer toolkit - cryptographic utility
```

So we can see that the version (`3.0.2-0ubuntu1.9`) will have the fixes.

Given that this is a false positive due to CVE scanners not understanding how Canonical maintains patch lines of `openssl` (and other libraries), I'm going to close this issue.

**Comment by xtremerui** [comment]
Thank you @robdimsdale for the details!
